vlasovsolver/velocity_space_diffusion_cart.cpp

```diff
@@ -0,0 +1,449 @@
+/*
```

Comment:
I guess this file (and the _mu variant) are no longer used? So they shouldn't be in this PR.